diff --git a/node_modules/react-scripts/config/webpack.config.js b/node_modules/react-scripts/config/webpack.config.js
index b7cf765..b8e9449 100644
--- a/node_modules/react-scripts/config/webpack.config.js
+++ b/node_modules/react-scripts/config/webpack.config.js
@@ -37,6 +37,7 @@ const eslint = require('eslint');
 const getCacheIdentifier = require('react-dev-utils/getCacheIdentifier');
 // @remove-on-eject-end
 const postcssNormalize = require('postcss-normalize');
+const FaviconsWebpackPlugin = require('favicons-webpack-plugin');

 const appPackageJson = require(paths.appPackageJson);

@@ -606,6 +607,32 @@ module.exports = function(webpackEnv) {
             : undefined
         )
       ),
+      isEnvProduction && new FaviconsWebpackPlugin({
+        logo: './logo.svg',
+        prefix: '',
+        mode: 'webapp',
+        favicons: {
+          appName: "Musik",
+          appShortName: "Musik",
+          background: "#636363",
+          theme_color: "#ffffff",
+          display: "standalone",
+          orientation: "any",
+          scope: "/",
+          start_url: "/",
+          pixel_art: false,
+          icons: {
+              android: true,
+              appleIcon: { offset: 8 },
+              appleStartup: true,
+              coast: true,
+              favicons: true,
+              firefox: true,
+              windows: { offset: 16 },
+              yandex: true
+          }
+        }
+      }),
       // Inlines the webpack runtime script. This script is too small to warrant
       // a network request.
       // https://github.com/facebook/create-react-app/issues/5358
